odoo/tests/common.py

```diff
@@ -351,8 +351,7 @@ def _format_message(records, expected_values):
                 self.fail(msg + _format_message(records, expected_values))
 
     def shortDescription(self):
-        doc = self._testMethodDoc
-        return doc and ' '.join(l.strip() for l in doc.splitlines() if not l.isspace()) or None
+        return None
```

Comment:
For the record, [returning None causes the test runner](https://docs.python.org/3/library/unittest.html#unittest.TestCase.shortDescription) to only log the test name, without any fancy/irrelevant descriptions, which was causing useless multi-line entries.